### DISCUSSION TITLE
Welcome to gilthoniel Discussions!

### DISCUSSION THREAD

**Discussion by Nuntis-Spayz** [Nuntis-Spayz/gilthoniel#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Darth-Flufficus** [comment]
Good day. My name is Rob. I am a LudoSport  athlete and instructor. I purchased two EVO animas for a saber staff build. currently, to start the staff, I have to hold the saber with one hilt up, initiate that blade when it flashes, then turn the staff over and repeat for the other hilt. Is it possible to recalibrate the accelerometer of one anima so that both read and initiate off of one motion? We did this in the old animas by recalibrating the accelerometer with one hilt held upside down. Thanks for any advice.

**Comment by Crocmag** [comment]
Hello,

I have question, i started this years, buy my first saber. I see your soft work only with "Anima EVO".
My question are : 
"Anima NXT" are compatible ?
In some time ?

And more generally : 
did you imagine a Debian version ?
An Android APP ?

Thank's for your work, your time.
